src/dataset-builder/SummaryCharts.ts

```diff
@@ -0,0 +1,150 @@
+export function generateAgeChartOptions() {
+  return {
+    categories: [
+      { short: '0-17' },
+      { short: '18-29' },
+      { short: '30-39' },
+      { short: '40-49' },
+      { short: '50-59' },
+      { short: '60-69' },
+      { short: '70-79' },
+      { short: '80+' },
+    ],
+    series: [{ data: [9, 25, 15, 17, 14, 9, 6, 5] }],
+    title: 'Age',
+    yTitle: 'OVERALL PERCENTAGES',
+    height: '400rem',
+    legendEnabled: false,
+    showSeriesName: false,
+    type: 'bar',
+  };
+}
+
+export function generateGenderChartOptions() {
+  return {
+    categories: [
+      { short: 'Man' },
+      { short: 'Woman' },
+      { short: 'Nonbinary' },
+      { short: 'Transgender' },
+      { short: 'Other' },
+      { short: 'Prefer not to Say' },
+    ],
```

Comment:
these are different gender distinctions than what are used on the cohort editor page, again we might want them to match, but since its mocked data it probably doesn't matter too much at this point bc if we unmock this we would just use the distinctions that exist in the data.

Reply:
(ditto above)